test/Core.Test/SecretsManager/Commands/EnableAccessSecretsManager/EnableAccessSecretsManagerCommandTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Repositories;
+using Bit.Core.SecretsManager.Commands.EnableAccessSecretsManager;
+using Bit.Test.Common.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
+using Bit.Test.Common.Helpers;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Core.Test.SecretsManager.Commands.EnableAccessSecretsManager;
+
+[SutProviderCustomize]
+public class EnableAccessSecretsManagerCommandTests
+{
+    [Theory]
+    [BitAutoData]
+    public async Task EnableUsers_UsersAlreadyEnabled_DoesNotCallRepository(
+        SutProvider<EnableAccessSecretsManagerCommand> sutProvider, ICollection<OrganizationUser> data)
```

Comment:
`data` is not very descriptive, I suggest keeping the same name as the parameter in the method under test. i.e. `organizationUsers`. Then it's immediately obvious what arguments you're arranging and asserting.